source/extensions/omni.isaac.lab/omni/isaac/lab/sim/simulation_context.py

```diff
@@ -464,11 +472,7 @@ def render(self, mode: RenderMode | None = None):
                 self.set_setting("/app/player/playSimulations", True)
         else:
             # manually flush the fabric data to update Hydra textures
-            if self._fabric_iface is not None:
```

Comment:
Hm this would break non env workflows where users didn't need to call a forward function. Maybe we add a flag on whether forward was called and in-case it wasn't, we do it implicitly?

Reply:
This shouldn't actually change any logic right? It's just moving a piece of the code into a method.